examples/twitter-connector/src/main/java/com/salesforce/einsteinbot/twitter/connector/service/EinsteinBotService.java

```diff
@@ -0,0 +1,101 @@
+import com.salesforce.einsteinbot.sdk.client.RequestHeaders;
+import com.salesforce.einsteinbot.sdk.model.AnyRequestMessage;
+import com.salesforce.einsteinbot.sdk.model.RequestEnvelope;
+import com.salesforce.einsteinbot.sdk.model.ResponseEnvelope;
+import com.salesforce.einsteinbot.twitter.connector.listener.TweetCreateResponseApiCallback;
+import com.salesforce.einsteinbot.twitter.connector.util.MessageTransformer;
+import com.twitter.clientlib.ApiException;
+import com.twitter.clientlib.api.TwitterApi;
+import com.twitter.clientlib.model.CreateTweetRequest;
+import com.twitter.clientlib.model.Tweet;
+import java.util.Arrays;
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+/**
+ * This class serves as the service layer to talk to EinsteinBots.
+ */
+@Service
+public class EinsteinBotService {
+
+  private static final Logger logger = LoggerFactory.getLogger(EinsteinBotService.class);
+
+  @Autowired
+  private ChatbotClient chatbotClient;
+
+  @Autowired
+  private TwitterApi twitterApi;
+
+  @Value("${twitter.user.name}")
+  private String twitterUserName;
+
+  @Value("${sfdc.einstein.bots.orgId}")
+  private String orgId;
+
+  @Value("${sfdc.einstein.bots.botId}")
+  private String botId;
+
+  @Value("${sfdc.einstein.bots.force-config-endpoint}")
+  private String forceConfigEndpoint;
+
+  private RequestHeaders requestHeaders;
+
+  private TweetCreateResponseApiCallback tweetCreateResponseApiCallback;
+
+  @PostConstruct
+  public void setup() {
+    this.requestHeaders = RequestHeaders
+        .builder().orgId(this.orgId)
+        .build();
+    this.tweetCreateResponseApiCallback = new TweetCreateResponseApiCallback();
+    logger.info("Setup EinsteinBotService");
+  }
+
+  /**
+   * Receives tweet and sends it to einstein bots for processing. Once it receives response from
+   * Einstein bots it replies back to that tweet. Currently it only consumes text from tweet and
+   * replies back with a text as well.
+   * <p>
+   *
+   * @param tweet
+   */
+  @Async
+  public void sendMessage(final Tweet tweet) {
+    logger.debug("Received tweet with text: {}", tweet.getText());
+    // 1. Convert tweet to einstein bot request envelope
+    AnyRequestMessage message = MessageTransformer.buildChatbotMessage(tweet, this.twitterUserName);
+    RequestEnvelope requestEnvelope = MessageTransformer.buildRequestEnvelope(
+        tweet.getConversationId(), this.botId, this.forceConfigEndpoint,
+        Arrays.asList(message));
+    logger.debug("Converted tweet into Einstein bot request envelope: {}", requestEnvelope);
+    // 2. Send message to einstein bot
+    ResponseEnvelope responseEnvelope = this.chatbotClient.sendChatbotRequest(requestEnvelope,
```

Comment:
Can we enclose this in try/catch and log exception? It's useful when Bot API return's non 200 resp?